cfeScraping/index.js

```diff
@@ -98,17 +97,25 @@ const tableToArrays = (selector) => {
   // eslint-disable-next-line comma-dangle
 };
 
+/**
+ * Get the number of rows in the table
+ * @param {string} selector
+ * @returns int
+ */
+
+const getRows = (selector) => {
+  const rows = document.querySelector(sel - ector).innerText;
```

Comment:
There is a Typo in querySelector param `sel - ector`

Reply:
I was testing this, and I forgot correct it